scripts/parse_tools/xml_tools.py

```diff
@@ -282,29 +254,19 @@ def write(self, file, encoding="us-ascii", xml_declaration=None,
               default_namespace=None, method="xml",
               short_empty_elements=True):
         """Subclassed write method to format output."""
-        if PY3 and (PYSUBVER >= 4):
-            if PYSUBVER >= 8:
-                et_str = ET.tostring(self.getroot(),
-                                     encoding=encoding, method=method,
-                                     xml_declaration=xml_declaration,
-                                     default_namespace=default_namespace,
-                                     short_empty_elements=short_empty_elements)
-            else:
-                et_str = ET.tostring(self.getroot(),
-                                     encoding=encoding, method=method,
-                                     short_empty_elements=short_empty_elements)
-            # end if
-        else:
+        if PYSUBVER >= 8:
             et_str = ET.tostring(self.getroot(),
-                                 encoding=encoding, method=method)
-        # end if
-        if PY3:
-            fmode = 'wt'
-            root = str(et_str, encoding="utf-8")
+                                    encoding=encoding, method=method,
+                                    xml_declaration=xml_declaration,
+                                    default_namespace=default_namespace,
+                                    short_empty_elements=short_empty_elements)
         else:
```

Comment:
does this need to be an "else if PYSUBVER >= 4"?

Reply:
I don't believe so.  Since 3.7 is the minimum, we should use the extended API for 3.8+, otherwise, the only way we should get into the `else` is on 3.7.

Reply:
Ah, yes. Forgot that we're cutting things off at 3.7